main.js

```diff
@@ -6,15 +6,17 @@ const sgAutorelease = require('./sg-auto-release');
 
 program
   .version(packageJson.version)
-  .option('-r, --repo [type]', 'Owner Name/Repo Name example: wix/wix-style-react')
-  .option('-s, --source-directory [type]', 'static files directory')
-  .option('-b, --root-path [type]', 'Root path to build agent root directory')
-  .option('-p, --pr [type]', 'Pull request number')
-  .option('-t, --github-token [type]', 'Github authentication token')
+  .option('--repo [value]', 'Owner Name/Repo Name example: wix/wix-style-react')
```

Comment:
All of these are breaking changes.
I suggest making it backward compatible.
Just add the new options instead of removing the old ones and map in the old to new.